snap/snapcraft.yaml

```diff
@@ -215,8 +215,8 @@ parts:
   containerd:
     plugin: make
     source: https://github.com/containerd/containerd.git
-    # from https://github.com/moby/moby/releases/tag/v24.0.5
-    source-tag: v1.6.21
+    # from https://github.com/moby/moby/releases/tag/v27.2.0
```

Comment:
For consistency if you apply other comments: https://github.com/moby/moby/blob/v27.2.0/Dockerfile#L199